cli/tests/e2e/snapshots/test_ci/test_outputs/noautofix---json/results.txt

```diff
@@ -390,8 +384,7 @@ SEMGREP_APP_TOKEN="fake_key" SEMGREP_USER_AGENT_APPEND="pytest" SEMGREP_SETTINGS
             }
           }
         },
-        "severity": "WARNING",
-        "validation_state": "NO_VALIDATOR"
```

Comment:
This is maybe wrong. Maybe you used an old version of semgrep or develop or semgrep-core.
The snapshot should still have those validation_state I think.
cc @akuhlens